src/static/assets/css/markdown.css

```diff
@@ -0,0 +1,695 @@
+@font-face {
```

Comment:
I would prefer it if we didn't store external code in our repo, but I think that is outside the scope of this PR. Would you create a story to address this?